module-db/tests/Helpers.hpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <Database/Database.hpp>
+#include <string>
+#include <filesystem>
+
+namespace db::tests
+{
+    std::filesystem::path getScriptsPath();
+    std::filesystem::path getPurePhoneScriptsPath();
+
+    class DatabaseScripts
+    {
+      public:
+        DatabaseScripts(std::filesystem::path directory, std::vector<std::string> &&exclude);
+        bool operator()(Database &db);
+
+      private:
+        std::filesystem::path directory;
+        std::string extension;
+        std::vector<std::string> exclude;
+    };
+
+    template <typename Db>
+    class DatabaseUnderTest
+    {
+      public:
+        DatabaseUnderTest(std::filesystem::path directory,
+                          std::filesystem::path scriptsPath,
+                          bool withDevelopment = false)
+        {
+            construct(directory, scriptsPath, withDevelopment);
+        }
+        explicit DatabaseUnderTest(std::filesystem::path directory)
+        {
+            construct(directory, {}, false);
+        }
+
+        ~DatabaseUnderTest()
+        {
+            Database::deinitialize();
+        }
+
+        Db &get()
+        {
+            return *db;
+        }
+
+      private:
+        void construct(const std::filesystem::path &directory,
```

Comment:
Why this `construct()` function instead of implementing directly in the first constructor?

Reply:
It's due to the having two constructors taking different parameters :)

Reply:
I see :) But why not reuse the more generic constructor inside the less generic one?

Reply:
Ah, that's what you mean. I decided it will be clearer to implement it that way but of course I can change it.

Reply:
Fixed